bluesky/tests/test_callbacks.py

```diff
@@ -586,3 +587,47 @@ def event(self, doc):
     bc = BrokerChecker(('img',), db=db)
     RE.subscribe(bc)
     RE(count([hw.direct_img]))
+
+
+def test_plotting_hints(RE, hw, db):
+    ''' This tests the run and checks that the correct hints are created.
+        Hints are mainly created to help the BestEffortCallback in plotting the
+        data.
+        Use a callback to do the checking.
+    '''
+    class HintChecker(CallbackBase):
```

Comment:
It is not documented, but https://github.com/NSLS-II/bluesky/blob/master/bluesky/tests/utils.py#L30 is helpful for this sort of test.

Reply:
great, let's use this. ill replace `HintChecker`.